brainbeats-backend/Controllers/SampleController.cs

```diff
@@ -26,112 +33,136 @@ public class SampleController : ControllerBase {
         queryString = CreateVertexQuery("sample", sampleId, body, edges);
       } catch {
         return BadRequest("Malformed request");

```

Comment:
Would it be possible to return what was malformed about the request?

Reply:
Specific errors about what was malformed would require too many case handling for all the possibilities between controllers, and a generic response will need to suffice